test/test_sparse.py

```diff
@@ -255,6 +255,20 @@ def test_tensor(x, res):
         res = self.ValueTensor(3, 4, 5, 0)
         test_tensor(x, res)
 
+    def test_to_sparse(self):
+        shape = [10, 5, 19, 8]
+        for dim, dim_sz in enumerate(shape):
+            sparse_dims = dim + 1
+            for nnz in range(1, dim_sz):
```

Comment:
why it needs to iterate through range of nnz here?

Reply:
In testing, I found a bug that occurred when nnz was >0 so figured I'd include a range of nnz.  Can remove if you think its a performance drag or not really necessary.

Reply:
I see, I would just check two cases:

- nnz = 0
- nnz > 0

I don't think here the test needs to go through a range of nnz